integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -429,6 +444,9 @@ public int open(String path, FuseFileInfo fi) {
         // Assuming I will never wrap around (2^64 open files are quite a lot anyway)
         mNextOpenFileId += 1;
       }
+    } catch (FileDoesNotExistException e) {
```

Comment:
Invalid path can't be thrown here?

Reply:
Double checked in DefaultFileSystemMaster, Invalid path can be thrown here. Fixed, thanks